napari/_tests/test_with_screenshot.py

```diff
@@ -545,3 +545,39 @@ def test_blending_modes_with_canvas(make_napari_viewer):
     img2_layer.blending = 'minimum'
     screenshot = viewer.screenshot(canvas_only=True, flash=False)
     np.testing.assert_array_equal(screenshot[:, :, 0], np.minimum(img1, img2))
+
+
+@skip_local_popups
+def test_active_layer_highlight_visibility(make_napari_viewer):
+    viewer = make_napari_viewer(show=True)
+    viewer.window._qt_viewer.set_welcome_visible(False)
+
+    # take initial screenshot (full black/empty screenshot since welcome message is hidden)
+    launch_screenshot = viewer.screenshot(canvas_only=True, flash=False)
+    assert launch_screenshot[..., :-1].max() == 0
```

Comment:
How come we're not just asserting the whole array max is 0 here?

Reply:
ohhh no of course that's fine, perhaps just add a comment saying that we're ignoring the alpha value!